spark-jobs/src/main/scala/za/co/absa/enceladus/common/JobCmdConfig.scala

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.common
+
+import org.apache.spark.storage.StorageLevel
+import scopt.OptionParser
+import za.co.absa.enceladus.dao.auth.{InvalidMenasCredentialsFactory, MenasCredentialsFactory, MenasKerberosCredentialsFactory, MenasPlainCredentialsFactory}
+
+import scala.util.matching.Regex
+
+case class JobCmdConfig(args: Array[String] = Array.empty,
```

Comment:
I really like this JobCmdConfig as an object to pass around. But I think we need to fine-tune it more. Even though we have this object that could be passed around, we list all the parameters to methods like `jobCmdConfig.this, jobCmdConfig.that, jobCmdConfig.something, ...`